### PR TITLE
Get cap by address

### PR DESCRIPTION
**Pull request by halink0803**

### CODE REVIEW

**Comment by halink0803** [comment]
[Stat | implement "/daily-limit-by-address/:addr" API](https://trello.com/c/OMSuTpYh/214-stat-implement-daily-limit-by-address-addr-api)

File: http/permission.go [hunk 1]
```diff
@@ -3,8 +3,9 @@ package http
...
 const (
-	ReadOnlyPermission    Permission = iota // can only read data
-	RebalancePermission                     // can do everything except configure setting
-	ConfigurePermission                     // can read data and configure setting, cannot set rates, deposit, withdraw, trade, cancel activities
-	ConfirmConfPermission                   // can read data and confirm configuration proposal
+	ReadOnlyPermission      Permission = iota // can only read data
+	RebalancePermission                       // can do everything except configure setting
+	ConfigurePermission                       // can read data and configure setting, cannot set rates, deposit, withdraw, trade, cancel activities
+	ConfirmConfPermission                     // can read data and confirm configuration proposal
+	UserDashboardPermission                   // can push kyc info to reserve stat
```
Comment (favadi) [inline http/permission.go]: Please reuse the ConfirmConfPermission permission for the new KYC API, as the consumer is already used it for the `/update-user-addresses` API.

File: stat/reserve_stat.go [hunk 2]
```diff
@@ -390,18 +395,43 @@ func (self ReserveStats) Stop() error {
...
+//GetTxCapByAddress return user Tx limit by wei
+func (rs ReserveStats) GetTxCapByAddress(addr ethereum.Address) (*big.Int, error) {
+	address := common.AddrToString(addr)
+	kyced, err := rs.GetKYCByAddress(address)
+	if err != nil {
+		return nil, err
+	}
+	var cap float64
+	if kyced {
+		cap = common.KycedCap().TxLimit
```
Comment (favadi) [inline stat/reserve_stat.go]: cap is a builtin keyword, could you please change the variable to something else?

File: http/permission.go (hunk 1 above)
Comment (halink0803) [reply http/permission.go]: @favadi I removed in next commit. However, as I checked again with user dashboard, they only call this api for core and have not use any key to call any api for core before. So, please confirm if this key is ConfirmConfPermission.

**Comment by halink0803** [comment]
There is another PR https://github.com/KyberNetwork/reserve-data/pull/412 was merged into master so I will close this PR.

**Pull request closed without merging**